test/e2e/tests/buyflow.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   '@tags': ['buyflow'],
   'Show offers on web': function (browser) {
     const setup = browser.page.setup();
-    setup.navigate().select('local');
+    setup.navigate().select('local_min');
```

Comment:
do you think it's be worth extracting these two lines into a common setUp() function that's shared between all the test cases in this file?

or since it's shared across all the e2e tests, maybe a testing utility that selects the environment (local vs local_min, etc)? I'm not sure if it's worth since it's only two lines, but perhaps we'll have more common test setup in the future.

Reply:
Good call! I added a `useMinifiedJs` command, and added it to a `beforeEach` method in each test suite